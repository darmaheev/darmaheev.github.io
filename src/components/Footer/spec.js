import React from "react";
import Footer from "./index";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

it("renders footer", () => {
  const footer = shallow(<Footer />);
  expect(footer).toMatchSnapshot();
});