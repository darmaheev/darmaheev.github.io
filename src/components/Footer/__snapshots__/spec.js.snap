// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FooterBody />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <ForwardRef>
          An open-source site by Valery Darmakheev, with help from Pokéapi.
        </ForwardRef>,
        <ForwardRef>
          All content is © Nintendo, Game Freak, and The Pokémon Company.
        </ForwardRef>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "An open-source site by Valery Darmakheev, with help from Pokéapi.",
        },
        "ref": null,
        "rendered": "An open-source site by Valery Darmakheev, with help from Pokéapi.",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bwzfXH",
            "isStatic": true,
            "rules": Array [
              "
  margin: 0;
  font-size: var(--footer-font-size);
  color: var(--footer-color);
  font-weight: normal;
",
            ],
          },
          "displayName": "styled.h1",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bwzfXH",
          "target": "h1",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "All content is © Nintendo, Game Freak, and The Pokémon Company.",
        },
        "ref": null,
        "rendered": "All content is © Nintendo, Game Freak, and The Pokémon Company.",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bwzfXH",
            "isStatic": true,
            "rules": Array [
              "
  margin: 0;
  font-size: var(--footer-font-size);
  color: var(--footer-color);
  font-weight: normal;
",
            ],
          },
          "displayName": "styled.h1",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bwzfXH",
          "target": "h1",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
    ],
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  height: var(--footer-height);
  background-color: var(--footer-background-color);
  border-top-color: var(--footer-border-top-color);
  border-top-width: 1px;
  padding: 0 30px;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <ForwardRef>
            An open-source site by Valery Darmakheev, with help from Pokéapi.
          </ForwardRef>,
          <ForwardRef>
            All content is © Nintendo, Game Freak, and The Pokémon Company.
          </ForwardRef>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "An open-source site by Valery Darmakheev, with help from Pokéapi.",
          },
          "ref": null,
          "rendered": "An open-source site by Valery Darmakheev, with help from Pokéapi.",
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bwzfXH",
              "isStatic": true,
              "rules": Array [
                "
  margin: 0;
  font-size: var(--footer-font-size);
  color: var(--footer-color);
  font-weight: normal;
",
              ],
            },
            "displayName": "styled.h1",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bwzfXH",
            "target": "h1",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "All content is © Nintendo, Game Freak, and The Pokémon Company.",
          },
          "ref": null,
          "rendered": "All content is © Nintendo, Game Freak, and The Pokémon Company.",
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bwzfXH",
              "isStatic": true,
              "rules": Array [
                "
  margin: 0;
  font-size: var(--footer-font-size);
  color: var(--footer-color);
  font-weight: normal;
",
              ],
            },
            "displayName": "styled.h1",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bwzfXH",
            "target": "h1",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
      ],
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  height: var(--footer-height);
  background-color: var(--footer-background-color);
  border-top-color: var(--footer-border-top-color);
  border-top-width: 1px;
  padding: 0 30px;
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
